specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/virtualclusters.json

```diff
@@ -0,0 +1,453 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.0",
```

Comment:
For other swagger files in this directory, this version is set to: "2015-05-01-preview". I'm not sure why this one was generated with "2.0" and whether I should change it to "2015-05-01-preview".

Reply:
Discussed offline, this value is important so we do need the correct version here :)

Reply:
Version changed, thanks.